packaging_automation/tests/test_package_cloud_statistics_collector.py

```diff
@@ -20,15 +24,28 @@
 def test_fetch_and_save_package_cloud_stats():
     db = create_engine(db_connection_string(db_params=db_parameters, is_test=True))
     db.execute(text(f'DROP TABLE IF EXISTS {PackageCloudDownloadStats.__tablename__}'))
-    pack_count = package_count(ORGANIZATION, REPO, PACKAGE_CLOUD_API_TOKEN)
     session = db_session(db_params=db_parameters, is_test=True)
     page_record_count = 3
     parallel_count = 3
+
+    filtered_package_count = get_filtered_package_count(session)
+
     for index in range(0, parallel_count):
         fetch_and_save_package_cloud_stats(package_cloud_api_token=PACKAGE_CLOUD_API_TOKEN, organization=ORGANIZATION,
                                            repo_name=REPO, db_params=db_parameters, parallel_count=parallel_count,
                                            parallel_exec_index=index, page_record_count=page_record_count,
                                            is_test=True, save_records_with_download_count_zero=True)
 
     records = session.query(PackageCloudDownloadStats).all()
-    assert len(records) == pack_count * PACKAGE_SAVED_HISTORIC_RECORD_COUNT
+
+    assert len(records) == filtered_package_count * PACKAGE_SAVED_HISTORIC_RECORD_COUNT
+
+
+def get_filtered_package_count(session) -> int:
+    result = stat_get_request(
+        package_list_with_pagination_request_address(PACKAGE_CLOUD_API_TOKEN, 1, ORGANIZATION, REPO, 100),
```

Comment:
Can you put a small comment explaining we can look at only the first page because we have less than 100 test packages

Reply:
added